tests/integration/test_documentation_validation.py

```
@@ -338,12 +409,29 @@
+        """Test that file uses consistent line endings throughout."""
         with open(SUMMARY_FILE, 'rb') as f:
             content = f.read()
-        
-        has_crlf = b'\r\n' in content
-        has_lf_only = b'\n' in content and b'\r\n' not in content
-        
-        assert has_lf_only or has_crlf, "File should have consistent line endings"
-        assert not (has_lf_only and has_crlf), "File should not mix line ending styles"
+
+        # Split preserving line endings and detect per-line endings
+        lines = content.splitlines(True)
```

Ruff (0.14.7)

Boolean positional value in function call

(FBT003)